AudioLink/Scripts/AudioLinkController.cs

```diff
@@ -64,7 +67,21 @@ void Update()
 
             void Start()
             {
-                if (audioLink == null) Debug.Log("Controller not connected to AudioLink");
+                if (audioLink == null) {
```

Comment:
I won't block the PR because of it, but again, nice to keep the bracing style consistent.

Reply:
Fixed. I'm too used to having a linter auto-fix my formatting ^^